src/tests.c

```diff
@@ -7086,11 +7086,14 @@ int main(int argc, char **argv) {
     run_context_tests(0);
     run_context_tests(1);
     run_scratch_tests();
+
     ctx = secp256k1_context_create(SECP256K1_CONTEXT_SIGN | SECP256K1_CONTEXT_VERIFY);
-    if (secp256k1_testrand_bits(1)) {
+    /* Randomize the context only with probability 15/16
+       to make sure we test without context randomization from time to time.
+       TODO Reconsider this when recalibrating the tests. */
+    if (!secp256k1_testrand_bits(4)) {
```

Comment:
I think this implements randomization in 1/16 of the cases.

Reply:
Ha, this happens if you want to do a "quick" change...

Fixed.

> Also, it randomizes with uninitialized memory?

Yeah, don't ask me why I removed this line.